From 1119ec2ef3aeb5ed798a4f72a3b7471a6b82754b Mon Sep 17 00:00:00 2001
From: Z User <z@container>
Date: Tue, 19 May 2026 10:50:12 +0000
Subject: [PATCH 1/2] a6eb10c8-3c09-45cc-900b-68b873c67918

---
 prisma/schema.prisma | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/prisma/schema.prisma b/prisma/schema.prisma
index 544bc87..c1fe501 100755
--- a/prisma/schema.prisma
+++ b/prisma/schema.prisma
@@ -3,7 +3,7 @@ generator client {
 }
 
 datasource db {
-  provider = "postgresql"
+  provider = "sqlite"
   url      = env("DATABASE_URL")
 }
 
-- 
2.47.3

